crates/oxvg_path/src/command.rs

```diff
@@ -367,6 +344,33 @@ impl std::fmt::Display for Data {
     }
 }
 
+pub fn short_number<F>(n: F) -> String
```

Comment:
We'll want to use this in other places so move to `oxvg_utils`

When we do so, we'll want to add feature flags to `oxvg_utils` to prevent unnecessary dependencies for users of `oxvg_path`